dev/examples/Drag-external-handlers.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react"
+import { useRef } from "react"
+import {
+    motion,
+    useMotionValue,
+    useMotionTemplate,
+    useTransform,
+} from "@framer"
+
+const container = {
+    width: "50%",
+    height: 300,
+    background: "rgba(255,255,255,0.5)",
+    borderRadius: 20,
+    display: "flex",
+    justifyContent: "flex-start",
+    alignItems: "center",
+}
+
+const child: React.CSSProperties = {
+    width: 200,
+    height: 200,
+    borderRadius: 20,
+    pointerEvents: "none",
+}
+
+export const App = () => {
+    const ref = useRef()
+    const x = useMotionValue(0)
+    const y = useMotionValue(0)
+    const style = useMotionTemplate`translate3d(${x}px, ${y}px, 0)`
+    const color = useTransform(
+        x,
+        [0, 100, 200, 300],
+        ["red", "yellow", "green", "blue"]
+    )
+    return (
+        <motion.div
```

Comment:
Some weird things happening in this example @InventingWithMonster , for some reason the drag constraints are resolving to have the same min and max values for both axis. Have I set something up wrong in this example?

Reply:
I'm surprised because I shouldn't have touched constraint generation 🤔